docs/HOW_LINES_WORK.md

```diff
@@ -0,0 +1,31 @@
+# What does "Lines" mean?
+
+A `line` is the shorthand term for a scoped channel through which an abortion fund
+communicates with a patient. It functions as a pipeline so that funds can keep patients
+separated if they so choose.
+
+For example, DCAF has separate phone lines for DC, Maryland, and Virginia; these lines have
+different case managers on duty and different pools of money. They're separated into their
+own lines so patients in a given service area are easier to keep track of.
+
+A case manager can only be working on one line at a time; their line scopes their search
+results.
+
+## What this means for engineers
+
+In [config/initializers/env_var_constants.rb](../config/initializers/env_var_constants.rb),
+we set an array of the lines `LINES`. (We default it to the DCAF use case, which operates
+lines for DC, MD, and VA.) It reads from the environment variable `DARIA_LINES`.
+
+We bubble that up and make it accessible in the app via the [LinesHelper](../app/helpers/lines_helper.rb).
+We also have a convenience method, `current_line`, which aliases to the session variable.
+
+Case managers have their lines set as a session variable, and they have to log out to
+change it.
+
+## What this means for abortion funds
```

Comment:
I suggest this section go above the engineer section, as technical users are interested in both but non-technical users only want to read through the intro and this one.

Reply:
clutch insight! Done